lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1204,9 +1204,10 @@ def _build_files_manifest_walk(b_collection_path, namespace, name, ignore_patter
     manifest = _make_manifest()
 
     def _walk(b_path, b_top_level_dir):
+        common_prefix = os.path.commonprefix([b_top_level_dir, b_path])
```

Comment:
looks like it is better to use commonpath instead of commonprefix as commonpath always returns a valid path

Reply:
btw @webknjaz thanks for the quick review!

Reply:
Thanks! Marking this thread as resolved.